vcr/vcr.go

```diff
@@ -125,6 +131,9 @@ func (c *vcr) Configure(config core.ServerConfig) error {
 	// store config parameters for use in Start()
 	c.config.datadir = config.Datadir
 
+	// copy strictmode for openid4vci usage
+	c.config.strictmode = config.Strictmode
```

Comment:
I find this pattern a bit ugly, mixing concerns of Config i.m.o. (Config is intended for unmarshalling/injecting config?)